src/app/zap-templates/zcl/data-model/chip/matter-devices.xml

```diff
@@ -2363,4 +2363,19 @@ limitations under the License.
             </include>
         </clusters>
     </deviceType>
+    <deviceType>
+        <name>Cook Surface</name>
+        <domain>CHIP</domain>
+        <typeName>Matter Cook Surface</typeName>
+        <class>Simple</class>
+        <profileId editable="false">0x0103</profileId>
+        <deviceId editable="false">0x0077</deviceId>
+        <clusters lockOthers="true">
+            <include cluster="Descriptor" client="false" server="true" clientLocked="true" serverLocked="true">
+            </include>
+            <include cluster="Temperature Control" client="false" server="true" clientLocked="true" serverLocked="true"></include>
+            <include cluster="Temperature Measurement" client="false" server="true" clientLocked="true" serverLocked="true">
```

Comment:
This requires both of those clusters, no?  But the spec says at least one has to be supported, not necessarily both.

Reply:
Did my latest change fix this @bzbarsky-apple ? If not, what should I change this to? Or if there is another example I can relate it to.
My latest change was to use the same methodology that was used from the refrigerator device.

Reply:
You want serverLocked="false" for both of the optional things.

Reply:
Updated.